creme/compat/creme_to_sklearn.py

```diff
@@ -164,54 +198,102 @@ class Creme2SKLClassifier(Creme2SKLBase, sklearn_base.ClassifierMixin):
     """
 
     def __init__(self, estimator: base.Classifier):
-        self.estimator = estimator
 
-    def fit(self, X, y):
-        """Fits to an entire dataset contained in memory.
+        # Check the estimator is either a BinaryClassifier or a MultiClassifier
+        if not isinstance(estimator, (base.BinaryClassifier, base.MultiClassifier)):
+            raise ValueError('estimator is not a BinaryClassifier nor a MultiClassifier')
 
-        Parameters:
-            X (array-like of shape (n_samples, n_features))
-            y (array-like of shape n_samples)
+        self.estimator = estimator
 
-        Returns:
-            self
+    def _more_tags(self):
 
-        """
+        # If converting a BinaryClassifier, notify that only binary classification is supported
+        return dict(binary_only=isinstance(self.estimator, base.BinaryClassifier))
```

Comment:
I'm not sure about this but you can leave it :). It's actually a good idea.

Reply:
That's to pass `sklearn`'s `check_estimator` tests. Otherwise it will pass in some example `y` with more than 2 classes, which may crash the underlying `creme` estimator. Actually it still does due to the bug I reference in the code but still it should be good to conform to `sklearn`'s [guidelines](https://scikit-learn.org/dev/developers/develop.html#estimator-tags).